cb-sol-cli/cmd/deploy.js

```diff
@@ -168,7 +169,7 @@ async function deployERC20Handler(args) {
     const factory = new ethers.ContractFactory(constants.ContractABIs.Erc20Handler.abi, constants.ContractABIs.Erc20Handler.bytecode, args.wallet);
 
 
-    const contract = await factory.deploy(args.bridgeContract, [], [], [], { gasPrice: args.gasPrice, gasLimit: args.gasLimit});
```

Comment:
what will happen if user will not provide --bridgeAddress but after calling deploy with (for example --all) nonce of the sender will be different, so  constants.BRIDGE_ADDRESS will hold wrong bridge address?

Reply:
If the user doesn't provide --bridgeAddress when deploying after a --all deploy or simply a --bridge deploy, the correct bridge address is still used by the next handlers to be deployed. I think this is due to the deterministic nature of the address constants and the bridge needing to be the first one for other handlers to be deployed. I just tested with some console logs to confirm this as well and the correct bridge address is used. 

I also debated simply setting a global variable in the file that is set upon deployment of the bridge contract, rather than storing the address within the passed in `args` object. This may be a much simpler and safer way to go about it.